.github/workflows/pr-comment-make-check.yml

```diff
@@ -0,0 +1,69 @@
+name: Trigger 'make check' on PR comment
+
+on:
+  issue_comment:
+    types: [created]
+
+permissions:
+  contents: read
+  pull-requests: read
+
+jobs:
+  trigger-make-check-on-comment:
+    if: |
+      github.event.issue.pull_request != null &&
+      (
+        contains(github.event.comment.body, 'jenkins test all') ||
+        contains(github.event.comment.body, 'jenkins test make check')
+      )
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check if comment author is a collaborator
+        id: check_user
+        run: |
+          comment_user="${{ github.event.comment.user.login }}"
+          repo="${{ github.repository }}"
+          status=$(curl -s -o /dev/null -w "%{http_code}" \
+            -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" \
+            "https://api.github.com/repos/$repo/collaborators/$comment_user")
+          if [ "$status" -ne 204 ]; then
+            echo "$comment_user is not a collaborator. Exiting."
+            echo "authorized=false" >> "$GITHUB_OUTPUT"
+            exit 0
+          fi
+          echo "authorized=true" >> "$GITHUB_OUTPUT"
+
+      - name: Exit if unauthorized
+        if: steps.check_user.outputs.authorized != 'true'
+        run: exit 0
+
+      - name: Extract PR number safely
+        id: extract_pr
+        run: |
+          pr_url="${{ github.event.issue.pull_request.url }}"
+          pr_number="${pr_url##*/}"
+          if ! [[ "$pr_number" =~ ^[0-9]+$ ]]; then
+            echo "Invalid PR number: $pr_number"
+            exit 1
+          fi
+          echo "GH_PULL_REQUEST_ID=$pr_number" >> "$GITHUB_ENV"
+
+      - name: Get PR SHA
+        id: pr_sha
+        run: |
+          pr_sha=$(curl -s -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" \
+            "${{ github.event.issue.pull_request.url }}" | jq -r .head.sha)
+          if ! [[ "$pr_sha" =~ ^[a-f0-9]{40}$ ]]; then
+            echo "Invalid PR SHA: $pr_sha"
+            exit 1
+          fi
+          echo "pr_sha=$pr_sha" >> "$GITHUB_OUTPUT"
+
+      - name: Trigger Jenkins job
+        run: |
+          curl -X POST "https://jenkins.ceph.com/job/ceph-pull-requests/buildWithParameters" \
+            --user "${{ secrets.JENKINS_USER }}:${{ secrets.JENKINS_API_TOKEN }}" \
+            --data-urlencode "GH_PULL_REQUEST_ID=${{ env.GH_PULL_REQUEST_ID }}" \
+            --data-urlencode "GH_PULL_REQUEST_SHA=${{ steps.pr_sha.outputs.pr_sha }}" \
+            --data-urlencode "TRIGGERED_BY=https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}"
```

Comment:
could we include, somehow, the fact that it was triggered by a comment, and maybe from who (since we have those up there)?  (It looks like TRIGGERED_BY is free-form, and, the more relevant info in front of eyes, the better)  I'm not sure what's in the run id, but if "triggered by a comment from <user>" is in that URL, it still saves a click.